test/common/ssl/ssl_socket_test.cc

```diff
@@ -3011,15 +3227,13 @@ TEST_P(SslSocketTest, SignatureAlgorithms) {
       TestEnvironment::substitute("{{ test_rundir }}/test/common/ssl/test_data/san_uri_key.pem"));
 
   // Connection using defaults (client & server) succeeds.
-  testUtilV2(listener, client, "", true, "", "", "spiffe://lyft.com/test-team", "", "",
-             "ssl.sigalgs.rsa_pss_rsae_sha256", "ssl.sigalgs.ecdsa_secp256r1_sha256", GetParam(),
```

Comment:
I know this is merged already, but... you removed the only thing that's tested here (`ssl.sigalgs.*` stats).

Reply:
@PiotrSikora Oops, let me put it back in a separate PR, sorry.

Reply:
@PiotrSikora Submitted an additional PR to fix it https://github.com/envoyproxy/envoy/pull/5429. Good catch!
I will go over all the tests and recheck the stats. More comments are welcome.